addOns/llm/src/main/java/org/zaproxy/addon/llm/services/LlmAssistant.java

```diff
@@ -0,0 +1,34 @@
+package org.zaproxy.addon.llm.services;
+
+import dev.langchain4j.service.SystemMessage;
+import dev.langchain4j.service.UserMessage;
+
+import dev.langchain4j.service.V;
+import org.zaproxy.addon.llm.communication.Confidence;
+import org.zaproxy.addon.llm.communication.HttpRequestList;
+
+public interface LlmAssistant {
+    @UserMessage("Given the following swagger generate list of chained HTTP request to simulate a real world user : {{swagger}} ")
+    HttpRequestList extractHttpRequests(String swagger);
+
+    @UserMessage("As software architect, and based on your previous answer, generate other potential missing endpoint that are not mentioned on the swagger file. For example, if there is GET /product/1, suggest DELETE /product/1 if it's not mentioned")
```

Comment:
endpoints plural?
in the file not on it?

Also even though the LLM may suggest them anyway in the end perhaps the prompt shouldn't include DELETE as the example? 😇

Reply:
Agree, but giving few shots (examples) can clarify more the task for an LLM.
More info here : https://www.promptingguide.ai/techniques/fewshot